boosters/node-beginner/src/api/controllers/repoController.js

```diff
@@ -26,9 +32,10 @@ const createRepo = async (req, res) => {
 
 const getRepo = async (req, res) => {
   const { repoId } = req.params;
-
+  const { userId } = req.body;
   const repo = await Repos.findByPk(repoId);
-  res.send(repo);
+
+  repo.userId === userId || repo.isPrivate === false ? res.send(repo) : res.send(null);
```

Comment:
Should it return a 401 if the request is not authorized?

Reply:
Probably - I was just following the graphQL one where null was  returned, but I think setting status code is better